### PR TITLE
docs: update some docs

### PR DESCRIPTION
**Pull request by archerny**
# Which issue does this PR close?

Closes #

# Rationale for this change
 
optimize documentation

# What changes are included in this PR?

reorganize the doc directories and add community role doc in English

# Are there any user-facing changes?

simplify the difficulty of community participation for English-speaking programmers

# How does this change test

no need

### CODE REVIEW

File: docs/community/ROLES.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# CeresDB community roles
+
+This document describes the roles in the CeresDB community, as well as the requirements and authority for each role.
+
+- [Summary](#Summary)
+- [Contributor](#contributor)
```
Comment (chunshao90) [inline docs/community/ROLES.md:6]: contributor -> Contributor

File: docs/community/ROLES.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+The requirements, responsibilities, and authorities for each role are described below.
+
+## Contributor
+
+**requirements**
+
+Submited a PR and merged in any official CeresDB project successfully.
+
+**responsibilities**
```
Comment (chunshao90) [inline docs/community/ROLES.md:28]: Responsibilities

File: docs/community/ROLES.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+- Committer
+- PMC
+
+The requirements, responsibilities, and authorities for each role are described below.
+
+## Contributor
+
+**Requirements**
+
+Submited a PR and merged in any official CeresDB project successfully.
```
Comment (waynexia) [inline docs/community/ROLES.md:25]:
```suggestion
Submit a PR and get merged in any official CeresDB project.
```

File: docs/community/ROLES-CN.md [hunk 4]
```diff
@@ -72,4 +70,4 @@
 
 - Pull Request review  权限
```
Comment (waynexia) [inline docs/community/ROLES-CN.md:71]:
```suggestion
- Pull Request review 权限
```

File: docs/community/ROLES.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# CeresDB community roles
+
+- [CeresDB community roles](#ceresdb-community-roles)
+    - [Summary](#summary)
+    - [Contributor](#contributor)
+    - [Committer](#committer)
+    - [PMC](#pmc)
```
[the remark this answers is not in the document]
Comment (archerny) [reply docs/community/ROLES.md]: fixed, it is now consistent with the CN version

**Review by waynexia — APPROVED** [review]

**Pull request merged**